pages/common/gh-repo.md

```diff
@@ -0,0 +1,20 @@
+# gh repo
+
+> Work with GitHub repositories on the command line.
+> More information: <https://cli.github.com/manual/gh_repo>.
+
+- Create a new repository (if the repository name is not set, it will default to the name of the current directory):
+
+`gh repo create {{name}}`
+
+- Clone a repository:
+
+`gh repo clone {{repository}}`
+
+- Fork and clone a repository:
+
+`gh repo fork {{owner}}/{{repository}} --clone=true`
```

Comment:
If you use just `--clone`, gh reads `true` implicitly, to use `false` you need to explicitly include it `--clone=false`.
I prefer to simplify the command example:
```suggestion
`gh repo fork {{owner}}/{{repository}} --clone`
```